src/lottie/index.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
```

Comment:
Please, put a copy of the license inside the lottie folder
https://github.com/Gamote/lottie-react/blob/main/LICENSE

```
The above copyright notice and this permission notice shall be
included in all copies or substantial portions of the Software.
```

And a README.md explaining a bit why did we include these sources and where the original repository is

Reply:
Done :)